Scripts/TopArea.gd

```diff
@@ -32,7 +37,7 @@ func _on_mouse_exited() -> void:
 	set_process_input(false)
 
 func _input(event: InputEvent) -> void:
-	if event is InputEventMouseButton:
+	if event is InputEventMouseButton and ProjectSettings.get_setting("display/window/size/borderless") == true:
```

Comment:
rather than doing it this way, would it be possible to completely turn off the _input function? `set_process_input(false)` that way we wouldn't need to process this at all, since the WM of the OS would take care of it.

Reply:
I did not exactly think of doing it but after testing on my machine, this seems to work!